.github/workflows/ci.yaml

```diff
@@ -129,33 +129,33 @@ jobs:
         include:
           # don't run against the following C* versions when using cassandra storage type
           - cassandra-version: "binary:1.2.19"
-            cucumber-options: '--tags ~@cassandra_2_1_onwards --tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards'
+            cucumber-options: '--tags ~@cassandra_2_1_onwards --tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
           - cassandra-version: "binary:2.0.17"
-            cucumber-options: '--tags ~@cassandra_2_1_onwards --tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards'
+            cucumber-options: '--tags ~@cassandra_2_1_onwards --tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
           - cassandra-version: 'binary:2.1.22'
-            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards'
+            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
           - cassandra-version: 'binary:2.2.19'
-            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards'
+            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
           - cassandra-version: 'binary:3.0.25'
-            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards'
+            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@cassandra_3_11_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
           - cassandra-version: 'binary:3.11.11'
-            cucumber-options: '--tags ~@cassandra_4_0_onwards'
+            cucumber-options: '--tags ~@cassandra_4_0_onwards --tags ~@metrics'
             experimental: false
             jdk-version: '8'
-          - cassandra-version: 'binary:4.0.1'
+          - cassandra-version: 'binary:4.0.1 --tags ~@metrics'
```

Comment:
I made a mistake in my example.
The `--tags`  argument should be in the `cucumber-options` matrix variable, not the `cassandra-version` one.
Hence, it should be the following:

```
          - cassandra-version: 'binary:4.0.1'
            cucumber-options: '--tags ~@metrics'
            experimental: false
            jdk-version: '11'
          - cassandra-version: 'github:apache/trunk'
            cucumber-options: '--tags ~@metrics'
            experimental: true
            jdk-version: '11'
```

Reply:
@van-vliet, I finally understand what's wrong. Trying to run multiple features in the same `mvn` call doesn't work because some resources don't seem to be cleaned up correctly.

Here's a new commit you can cherry pick which separates features in distinct calls and removes the metrics cucumber tag which isn't necessary after all: https://github.com/thelastpickle/cassandra-reaper/pull/1199/commits/b9a384c5e4093df81262b9403a4a6e81b085e73d